code/controllers/subsystem/research.dm

```diff
@@ -306,7 +306,11 @@ SUBSYSTEM_DEF(research)
 	for(var/A in subtypesof(/obj/item/seeds))
 		var/obj/item/seeds/S = A
 		var/list/L = list()
-		L[TECHWEB_POINT_TYPE_GENERIC] = 50 + initial(S.rarity) * 2
+		//First we get are yield and rarity and times it by two
+		//Then we subtract production and maturation, making it so faster growing plants are better for RnD
+		//Then we add in lifespan and potency,
+		//A basic seed can be worth 248 points if its the best it can be.
+		L[TECHWEB_POINT_TYPE_GENERIC] = 50 + initial(((S.rarity + S.yield) * 2) - S.maturation - S.production + S.lifespan + S.potency)
```

Comment:
Math here is
50 + (0 + 10) x 2) - 1 - 1 + 100 + 100
meaning 50 + 20 - 2 + 200
18 + 250 
268 points for a Min-Maxed plant that would likely need a DNA manip at t4 + disks to even be able to get.